docs/src/howto-guides/026_how_to_install_thin_edge_manually.md

```diff
@@ -0,0 +1,223 @@
+# How to install thin-edge manually with OpenRC
```

Comment:
```suggestion
# How to install thin-edge.io manually with OpenRC
```

I see `thin-edge` in other places. Better to use `thin-edge.io` instead as it's our official product name.